public/src/main/java/com/revenuecat/purchases/models/GoogleProrationMode.kt

```diff
@@ -32,6 +32,14 @@ enum class GoogleProrationMode(
      */
     IMMEDIATE_WITH_TIME_PRORATION(BillingFlowParams.ProrationMode.IMMEDIATE_WITH_TIME_PRORATION),
 
+    /**
+     * Replacement takes effect when the old plan expires, and the new price will be charged at the same time.
+     *
+     * Example: Samwise's Tier 1 subscription continues until it expires on April 30. On May 1st, the
+     * Tier 2 subscription takes effect, and Samwise is charged $36 for his new subscription tier.
+     */
+    DEFERRED(BillingFlowParams.ProrationMode.DEFERRED),
```

Comment:
This is a breaking change in a public enum and technically should be a major version. But we have to add the value and we can't do major version now... Maybe we could add it at the end of the enum, so the integer value of `IMMEDIATE_AND_CHARGE_FULL_PRICE` and `IMMEDIATE_AND_CHARGE_PRORATED_PRICE` doesn't change.

This is going to break switch statements, asking them to be exhaustive. I don't think it's a biggie since I don't think many developers are doing a switch on this, but by putting it at the end of the enum at least we don't break the values of `IMMEDIATE_AND_CHARGE_FULL_PRICE` and `IMMEDIATE_AND_CHARGE_PRORATED_PRICE`.

Reply:
Moved to the bottom.